component/main.jsonnet

```
@@ -59,6 +59,11 @@ local patchRemoteWrite(promConfig) = promConfig {
     function(name) transformRelabelConfigs(rwd[name] { name: name }),
     std.objectFields(rwd)
   ),
+} + {
+  remoteWrite: std.map(
+    function(rw) rw + com.makeMergeable(defaults),
```

IMO this should be 

```suggestion
    function(rw) defaults + com.makeMergeable(rw),
```
so that users can override the defaults for individual remote write configs if necessary.

---

Right 🤦